eo-runtime/src/main/eo/org/eolang/negative-infinity.eo

```diff
@@ -61,9 +61,15 @@
   # Multiplication of $ and x
   [x...] > times
     x.at 0 > first!
+
```

Comment:
@maxonfjvipon Is that blank line required?

Reply:
@volodya-lombrozo no, but I think it increases readability if we separate objects with an empty line